cmd/operator/deploy/operator/12-alertmanager.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: gmp-system
+  name: gmp-alertmanager
+data:
+  config.yaml: |
+    receivers:
+      - name: "noop"
+    route:
+      receiver: "noop"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  namespace: gmp-system
+  name: gmp-alertmanager
+spec:
+  selector:
+    app.kubernetes.io/name: gmp-alertmanager
+  clusterIP: None
+  ports:
+  - port: 9093
+    targetPort: 9093
+    name: alertmanager
+---
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  namespace: gmp-system
```

Comment:
@lyanco should the default alertmanager go in `monitoring` similar to https://github.com/GoogleCloudPlatform/prometheus-engine/pull/276? Users will need to be able to edit the configmap it mounts to configure it

Reply:
I think gmp-public is where AM and the config should go, that's where the operator config lives, and people are supposed to be able to edit that.

FWIW I thought of using gmp-public for #276 , however the kube-state-metrics manifest was using monitoring, and I deferred to the ghost of Fabian there (didn't know if there was a specific reason that it had to live in "monitoring" or not so I didn't want to touch it). So I moved node-exporter to monitoring instead of gmp-public to keep locally consistent. But if it wouldn't break anything, then I think they should all live in gmp-public.

Reply:
sounds good, I actually thought `gmp-public` at first before I saw the other PR. updated